wiz-outpost-lite/templates/deployment.yaml

```diff
@@ -25,6 +25,23 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
+      securityContext:
```

Comment:
This should be as simple as `{{ toYaml .Values.securityContext ... }}`, do what's necessary in calculating the values above to make it so